cdap-ui/app/cdap/components/Alert/index.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, {Component, PropTypes} from 'react';
+import {Modal} from 'reactstrap';
+require('./Alert.less');
+
+export default class Alert extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showAlert: false || props.showAlert,
+      message: props.message,
+      type: props.type
+    };
+  }
+  componentWillReceiveProps(nextProps) {
+    let {showAlert, type, message} = nextProps;
+    if (
+      showAlert !== this.state.showAlert ||
+      type !== this.state.type ||
+      message !== this.state.message
+    ) {
+      this.setState({
+        showAlert,
+        type,
+        message
+      });
+    }
+  }
+  render() {
+    return (
+      <Modal
```

Comment:
`Alert` in reactstrap is not something that is floating at the top of the window. Its Inline to the element as far as I understood.